src/models/spaces/breeding/breeding.ts

```diff
@@ -17,7 +40,12 @@ export const BreedingModel = types
     motherGamete: types.maybe(types.string),
     fatherGamete: types.maybe(types.string),
     rightPanel: types.optional(RightPanelTypeEnum, "instructions"),
-    instructions: ""
+    instructions: "",
+    showSexStack: false,
+    showHeteroStack: false,
+    interactionMode: types.optional(BreedingInteractionModeEnum, "none"),
```

Comment:
This is something to confirm with Frieda, but I believe we had discussed entering the space with interaction mode already on "breed." Otherwise the user has nothing they can do when they enter, unless they click on a mode.

Separately (but related) I think we might have decided that the user must always be in one of three modes here, so "none" might be removed, and clicking on a selected button does nothing.

Reply:
I set the default mode to "breed".  I'd like to check with Michael and others about ensuring that a mode is always set (since the mock-up implied that there could be a "none" state).  Might very well be something that appears in the mock-ups once they are updated.